cluster-autoscaler/cloudprovider/externalgrpc/protos/externalgrpc.pb.go

```diff
@@ -786,7 +786,7 @@ type GetAvailableGPUTypesResponse struct {
 	unknownFields protoimpl.UnknownFields
 
 	// GPU types passed in as opaque key-value pairs.
-	GpuTypes map[string]*anypb.Any `protobuf:"bytes,1,rep,name=gpuTypes,proto3" json:"gpuTypes,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
+	GpuTypes map[string]*any1.Any `protobuf:"bytes,1,rep,name=gpuTypes,proto3" json:"gpuTypes,omitempty" protobuf_key:"bytes,1,opt,name=key,proto3" protobuf_val:"bytes,2,opt,name=value,proto3"`
```

Comment:
Note that `any1` is imported from `"github.com/golang/protobuf/ptypes/any"`. `"github.com/golang/protobuf/ptypes/any"` internally uses `anypb`

https://github.com/kubernetes/autoscaler/blob/b569db410fd8bb152c4cc1a940f059c7e2bf0538/cluster-autoscaler/vendor/github.com/golang/protobuf/ptypes/any/any.pb.go#L4-L15